changelogs/fragments/1221-aws_codebuild-tagging.yml

```
@@ -0,0 +1,20 @@
+minor_changes:
+- aws_codebuild - add support for ``purge_tags`` parameter
+  (https://github.com/ansible-collections/community.aws/pull/1221).
+- aws_codebuild - the ``source`` and ``artifacts`` parameters are now optional unless creating a new project
+  (https://github.com/ansible-collections/community.aws/pull/1221).
+- aws_codebuild - add the ``resource_tags`` paramater which takes the dictionary format for tags instead of the list of dictionaries format
+  (https://github.com/ansible-collections/community.aws/pull/1221).
+- aws_codebuild - add the ``resource_tags`` return value which returns the standard dictionary format for tags instead of the list of
+  dictionaries format
+  (https://github.com/ansible-collections/community.aws/pull/1221).
+
+bugfixes:
+- aws_codebuild - fix bug where the result may be supuriously flagged as ``changed`` when multiple tags were set on the project
```

```suggestion
- aws_codebuild - fix bug where the result may be spuriously flagged as ``changed`` when multiple tags were set on the project
```

honestly have never heard of this word before but I think its a typo :)